src/util/Cache.h

```diff
@@ -407,15 +400,16 @@ class FlexibleCache {
   void removeOneEntry() {
     AD_CONTRACT_CHECK(!_entries.empty());
     auto handle = _entries.pop();
-    _totalSizeNonPinned -= _valueSizeGetter(*handle.value().value());
+    _totalSizeNonPinned =
+        _totalSizeNonPinned - _valueSizeGetter(*handle.value().value());
```

Comment:
Don't you have `operator-=` in the meantime?

Reply:
Done, see https://github.com/ad-freiburg/qlever/pull/1111/commits/aa91fc6876a02bdece67b4c9b687c8abf2a92296.